### ISSUE TITLE
NodeJS Interactive window stuck

### ISSUE THREAD

**Issue by kant2002** [Microsoft/nodejstools#578]
I run express application via `.npm start` and now I have to restart VS to stop that application from running. There no way to terminate session.

**Comment by mousetraps** [comment]
Yeah... We never quite implemented terminating sessions :-/. As a workaround for now... instead of restarting vs, you can kill the process in task manager.

**Comment by mhkubaid** [comment]
So what is the fix in VS2017?

**Comment by crazycrank** [comment]
Still no fix as it seems...

**Comment by jtcoyle** [comment]
I just ran into this same issue.  It would be nice to be able to CTL + C to exit the running nodejs client or server app from within the interactive window rather than to have a separate command window open outside of VS 2017..

**Comment by chris31389** [comment]
This pretty much makes using the npm interactive window useless to run applications. Back to using a normal Powershell terminal...
